dotnet/src/SemanticKernel/KernelExtensions/ImportSemanticSkillFromDirectory.cs

```diff
@@ -53,48 +53,51 @@ public static class ImportSemanticSkillFromDirectoryExtension
     /// </summary>
```

Comment:
This is a clever adaptation, but I wonder how intuitive the (original) call was. 

From the caller's perspective why do we have to specify a parent directory (that effectively means nothing) plus a subdirectory? Why not just provide a list of directories you want to load from?

The current approach forces all callers to have something like a "skills" folder, which doesn't really make any sense. If you just have one or two skills, and each of those is in a folder in the root, we should be able to just register those by passing their folder names directly. (Again, I know this predates this PR, but it's a good opportunity to fix it).

My suggestions:
 - keep this method signature, as I see the utility.
 - refactor out the part inside the while loop into another method for readability
 - make it so that, you can pass in a folder name to the "parentDirectory" arg, and that recursively searches for skills regardless throughout its descendants. The user could pass in the skill directory itself, a higher-level "skills" folder, or even the whole app -- we should discover the available skills under each.
 - the current "skillDirectoryNames" becomes more of a string filter, rather than looking for those directories specifically -- allows you to optionally specify which of the discovered skills to import. As skills are discovered in the folder descendants, it checks to see if their names are in the filtered list (or if filter list is empty) before registering.

Reply:
I think the suggestion is great but I opened this PR to be small and fast, not introducing a big change or greater improvement to the way this method behaves.